pyproject.toml

```diff
@@ -39,8 +39,8 @@ dependencies = [
 ]
 
 [tool.hatch.envs.default.scripts]
-test = "pytest {args:tests}"
-test-cov = "coverage run -m pytest {args:tests}"
+test = "pytest {args:tests tests/integration}"
+test-cov = "coverage run -m pytest {args:tests tests/integration}"
```

Comment:
For the coverage, it makes sense to have a single script that capture coverage for both. Let's leave it as-is.